src/Faker/Provider/es_MX/Person.php

```diff
@@ -95,21 +95,28 @@ public static function curp($firstName = null, $lastNameFather = null, $lastName
 
     public static function rfc($firstName = null, $lastNameFather = null, $lastNameMother = null, $birthDate = null, $gender = null)
```

Comment:
This really shouldn't be there, but since there are many helpers in Person class that needs to be in Payment class also, I couldn't figure out how to do it better, Maybe Trait, but not all PHP versions support Traits